testsuite/domain/src/test/resources/host-configs/host-slave.xml

```diff
@@ -82,7 +82,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
```

Comment:
Adding this:

        <server name="reload-two" group="reload-test-group"
 auto-start="false">
            +<socket-bindings socket-binding-group="reload-sockets"
port-offset="1350"/>
            <jvm name="default" />
        </server>

Causes reload-two to pick up the change and the test to pass. I'm looking
at why adding a socket group would be ignore now.


On Fri, Sep 11, 2015 at 3:41 PM, Ken Wills <ken@zaptillion.net> wrote:

> I've verifed this is failing, and it also fails if all of the other
> ignored-resources are commented out. I'm taking a closer look now.
>
> On Fri, Sep 11, 2015 at 3:23 PM, Brian Stansberry <
> notifications@github.com> wrote:
>
>> In testsuite/domain/src/test/resources/host-configs/host-slave.xml
>> <https://github.com/wildfly/wildfly-core/pull/1057#discussion_r39312956>:
>>
>> > @@ -82,7 +82,7 @@
>> >
>> >      <domain-controller>
>> >          <!-- Remote domain controller configuration with a host and port -->
>> > -        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
>> > +        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
>>
>> If I remove this, ServerRestartRequiredTestCase fails, but I don't
>> understand why. The test adds a bogus socket-binding config to a
>> socket-binding-group that should not be being ignored. Doing so does not
>> result in the expected failure on host=slave/server=reload-two.
>>
>> I'm concerned that if ignore-unused-configuration is undefined/true, the
>> slave is ignoring stuff it should not be ignoring. That's a simple
>> explanation for the test failure.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/wildfly/wildfly-core/pull/1057/files#r39312956>.
>>
>
>